lib/collection/src/collection/resharding.rs

```diff
@@ -0,0 +1,16 @@
+use serde::{Deserialize, Serialize};
+
+use crate::shards::shard::{PeerId, ShardId};
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct Progress {
+    pub peer_id: PeerId,
+    pub shard_id: ShardId,
+}
+
+impl Progress {
+    #[allow(dead_code)]
+    pub fn new(peer_id: PeerId, shard_id: ShardId) -> Self {
+        Self { peer_id, shard_id }
+    }
+}
```

Comment:
I assume this tracks the current state of the resharding progress. I'd prefer to call it `State` instead. Just because this looks like a structure that tracks progress to be reported somewhere, like the other progress trackers we have.

Reply:
Done.